mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/166 - View Type Dialogs

Closes #166 

To test:
- Play introductory scenario
- Make sure all views work properly in scenario (triggering correct help tips and everything)
- Make sure attack log button changes color at the right time and attack log can be opened on top of any other view
- Make sure close button on different views works correctly
